azext_iot/central/models/ga_2022_05_31/organization.py

```diff
@@ -0,0 +1,10 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+class Organization:
```

Comment:
Organization should have `parent` field as well.

Reply:
Sure. Thanks for catching.